src/Components/ArticleByTopic.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticlesFromApi } from "./Utils/api";
+
+const ArticleByTopic = (AllArticles) => {
+  const [articles, setArticles] = useState([]);
+  const { topic_slug } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+  const [err, setErr] = useState(null);
+
+  useEffect(() => {
+    getArticlesFromApi(topic_slug)
+      .then((articlesApi) => {
+        setArticles(articlesApi);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        setErr(`Not found`);
+      });
+  }, [topic_slug]);
+  if (err) return <p>{err}</p>;
+  if (isLoading) return <h1> loading</h1>;
+  return (
+    <div>
+      {articles
+        .filter((article) => article.topic === topic_slug)
+        .map((filteredPerson) => (
+          <li>
+            <b>{filteredPerson.title}</b>
+            <p>{filteredPerson.body}</p> {filteredPerson.topic},
```

Comment:
Is `{filteredPerson.topic},` supposed to still be in here? If so then I'd wrap it within a tag and clean it up a little as at the moment when you look at the list of all the comments it shows the topic with the author's name in bold on the same line